### PR TITLE
Layout fixes for NC14, closes #184

### LINKED ISSUE #184: Compatibility with Nextcloud 14
Any chance you can try to look into that @raimund-schluessler  ?

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This fixes the layout for NC14 but will most likely break layout for NC<14.
@skjnldsv @nickvergessen 

PS: The diff for templates/main.php is huge, because the indentation changed (I removed the outer div).

### CODE REVIEW

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/tasks/pull/185/files?w=1 is here to help with the diff, it ignores whitespace changes

**Comment by nickvergessen** [comment]
Didn't test, but from what I saw on other repos it should fix it

File: css/src/style.scss [hunk 1]
```diff
@@ -327,7 +326,7 @@ $white: #fff;
 		margin: 12px 0;
```
Comment (skjnldsv) [inline css/src/style.scss]: You ca use the new add button standard! :) 
https://github.com/nextcloud/server/blob/master/core/css/apps.scss#L99
```html 
<!-- app-navigation -->
<div class="app-navigation-new">
<button type="button" id="new-user-button" class="icon-add">New user</button>
</div>
<!-- ul -->
```

Comment (raimund-schluessler) [reply css/src/style.scss]: You mean to use the new button to create new tasks?

Comment (skjnldsv) [reply css/src/style.scss]: Yes. You can look the 14 documentation. On the design section. :)

**Pull request merged**